docs/install/windows-release-channels.md

```diff
@@ -0,0 +1,320 @@
+- **Switching channels** (Stable ↔ Nightly) - Recommended
+- **Troubleshooting broken installation** - Essential
+- **Upgrading within same channel** - Optional
+
+**When not to use REINSTALL=ALL:**
+- Fresh installations (no existing Netdata)
+- When you want minimal disruption (standard MSI upgrade is sufficient)
+
+:::warning
+
+The `REINSTALL=ALL` parameter ensures complete replacement of binaries but is not required for normal channel switches. The MSI installer handles upgrades automatically.
+
+:::
+
+## Verification: Confirm Your Channel
+
+After switching channels, verify which version you're running:
+
+<details>
+<summary><strong>Check via Netdata Dashboard</strong></summary><br/>
+
+1. Open your browser to `http://localhost:19999`
+2. Click the **Netdata logo** in the top-left corner
+3. Look for the version number:
+   - **Stable**: `v2.1.0` (clean version number)
+   - **Nightly**: `v2.1.0-123-nightly` (includes commit hash)
+
+</details>
+
+<br/>
+
+<details>
+<summary><strong>Check via PowerShell</strong></summary><br/>
+
+```powershell
+# Get installed Netdata version
+Get-ItemProperty "HKLM:\Software\Microsoft\Windows\CurrentVersion\Uninstall\*" | 
+  Where-Object { $_.DisplayName -like "*Netdata*" } | 
+  Select-Object DisplayName, DisplayVersion, Publisher
+```
+
+**Example output:**
+```
+DisplayName    : Netdata
+DisplayVersion : 2.1.0
+Publisher      : Netdata Inc.
+```
+
+</details>
+
+<br/>
+
+<details>
+<summary><strong>Check Netdata Service Status</strong></summary><br/>
+
+```powershell
+# Check Netdata service status
+Get-Service netdata | Select-Object Name, Status, DisplayName
+
+# Check service executable path
+Get-WmiObject win32_service | Where-Object {$_.Name -eq 'netdata'} | Select-Object PathName
+```
+
+</details>
+
+## Troubleshooting Common Issues
+
+<details>
+<summary><strong>Issue: "Another version is already installed"</strong></summary><br/>
+
+**Symptoms**: MSI installer shows error about existing installation
+
+**Solution**: Use the `REINSTALL=ALL` parameter explicitly:
+
+```powershell
+msiexec /qn /i netdata-x64.msi REINSTALL=ALL
+```
+
+Alternatively, uninstall first, then reinstall:
+
+```powershell
+# Uninstall existing installation
+msiexec /qn /x netdata-x64.msi
+
+# Wait for uninstall to complete
+Start-Sleep -Seconds 10
+
+# Reinstall with new channel
+msiexec /qn /i netdata-x64.msi TOKEN="<YOUR_TOKEN>" ROOMS="<YOUR_ROOMS>"
+```
+
+</details>
+
+<br/>
+
+<details>
+<summary><strong>Issue: Cloud Connection Lost After Switch</strong></summary><br/>
+
+**Symptoms**: After switching channels, node doesn't appear in Netdata Cloud
+
+**Solution**: Reclaim the node:
+
+```powershell
+# Stop Netdata service
+Stop-Service netdata
+
+# Remove old cloud configuration
+Remove-Item "C:\Program Files\Netdata\var\lib\netdata\cloud.d\*" -Force
+
+# Reinstall with claim token
+msiexec /qn /i netdata-x64.msi TOKEN="<YOUR_TOKEN>" ROOMS="<YOUR_ROOMS>" REINSTALL=ALL
+
+# Start service
+Start-Service netdata
```

Comment:
Like previous comment, it would be useful to also show the WIndows Services screen.